tailwind.config.cjs

```diff
@@ -1,7 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   mode: 'jit', // update this line
-  content: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
+  content: ['./src/**/*.{js,jsx,ts,tsx}', './index.html'],
```

Comment:
👍

Reply:
Yea just to be clear, tailwind prunes all classes not used by any of these files. So the classes I put in index.html that weren't used elsewhere were being pruned without this change, pretty cool!